frontend/apps/reader/modules/readerfont.lua

```diff
@@ -51,19 +52,46 @@ function ReaderFont:init()
     local face_list = cre.getFontFaces()
     for k,v in ipairs(face_list) do
         table.insert(self.face_table, {
-            text = v,
+            text_func = function()
+                -- defaults are hardcoded in credocument.lua
+                local default_font = G_reader_settings:readSetting("cre_font") or self.ui.document.default_font
+                local fallback_font = G_reader_settings:readSetting("fallback_font") or self.ui.document.fallback_font
+                local text = v
+                if v == default_font then
+                    text = text .. "   ★"
+                end
+                if v == fallback_font then
+                    text = text.."   ♻"
```

Comment:
Recycling? :-P

Reply:
Not much choice with what's available in Noto fonts (including the updated ones, seen no other symbols than the ones I'm used to).
[test_unichars.html.txt](https://github.com/koreader/koreader/files/746975/test_unichars.html.txt) : sample file where I do my icon shopping.

Reply:
There's a (couple of) relatively small NotoSymbols that we could pull if you fancy something in there... ;).

Reply:
I guess we should be fine with what we have.
Btw, the greek glyphs have been corrected in your updated Noto Sans CJK, and they have also been included in the main Noto Sans, so good idea to update them :)